desktop/checkRequiredLibs.ts

```diff
@@ -0,0 +1,147 @@
+import { promisify } from 'util';
+import { exec as execCallback } from 'child_process';
+import path from 'path';
+import fs from 'fs';
```

Comment:
```suggestion
import { access } from 'fs/promises';
```
And then you won't need to call `promisify(fs.access)` ;)